spec/features/user_sees_pagination_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+feature "pagination" do
+  context "visiting index page with many pieces" do
+    before do
+      visit new_piece_path
+      100.times do
+        piece = FactoryGirl.create(:piece)
+        fill_in 'Title', with: piece.title
+        fill_in 'Composer', with: piece.composer
+
+        click_button "Add Piece"
+      end
```

Comment:
Use `FactoryGirl`'s `create_list` to create 100 pieces, otherwise you're also testing your add new piece functionality, and in general you don't want one test to depend on other tests.